src/components/app.js

```diff
@@ -74,9 +50,18 @@ class App extends React.Component {
             </div>
           </div>
           <div className='content-block'>
-            <div className='content-container'>
-              <Route exact path='/' component={Introduction} />
-              <Route path='/:componentKey' render={getComponent} />
+            <div className='content-container ds-l-row'>
+              <div className='ds-u-float--left ds-u-padding-right--6 ds-u-padding-top--2 ds-l-col--3'>
+                <ul className='ds-c-vertical-nav__subnav'>
+                  {LeftNav}
+                </ul>
+              </div>
+              <div className='ds-u-float--left ds-u-padding--1 ds-l-col--9'>
+                <div className='ds-u-measure--wide'>
+                  <Route exact path='/' component={Introduction} />
+                  <Route exact path='/:componentKey' render={ActiveComponent} />
```

Comment:
@kencheeto What does it add to enumerate each route?

Also - the links themselves are also grouped so the existing structure of `routes.js` is required in some form.